eslint.config.mjs

```diff
@@ -9,8 +13,6 @@ export default antfu(
       'ts/no-non-null-asserted-optional-chain': 'off',
       'ts/ban-ts-comment': 'warn',
       'ts/consistent-type-definitions': 'off',
-      'import/first': 'off',
-      'import/order': 'off',
```

Comment:
We need this 2 lines else it would mess with the way we export `provide/inject` from `vue` SFC

Reply:
Fixed 👍🏻